docs/dev-proc/source-code-structure.md

```diff
@@ -47,15 +39,15 @@ Dasharo Release tags in git repository use format: `<platform>_vX.Y.Z`
 
 ## New platform support
 
-Branch for new platform should be created from most recent`master` branch tag.
-If there is justified need to create support for new board at arbitrary
-non-tagged commit developer should mark this commit with `<platform>_v0.0.0`
-tag.
+Branch for new platform should be created from the most recent commit on the
+`dasharo` branch. If there is justified need to create support for new board at
+arbitrary non-tagged commit developer should mark this commit with
+`<platform>_v0.0.0` tag.
 
 ## Force-pushes rules
 
-Force-pushes to `<platform>/rel_vX.Y.Z`, `<platform>/develop` or
-`<platform>/<feature>` are forbidden with following exceptions:
+Force-pushes to `<platform>_rel_vX.Y.Z` of `<feature>` are forbidden with the
+following exceptions:
```

Comment:
Force pushes will only be forbidden for dasharo and master/main

Reply:
fixed in 1ed0a83f08334d132029bbed91dd088fea7ee020